tests/test_data/test_filter.py

```diff
@@ -37,11 +40,90 @@ def test_filter_with_empty_region_codes(self, sample_lazyframe):
 
         assert result.to_dicts() == expected.to_dicts()
 
-    def test_filter_with_no_matching_codes(self, sample_lazyframe):
+    def test_filter_with_no_matching_codes(self, sample_lazyframe: pl.LazyFrame):
         # Test with region codes that don't match any rows (should return an empty DataFrame)
         result = filter_sa1_regions(
             sample_lazyframe, ["999999"], "SA1_CODE_2021"
         ).collect()
 
         expected = pl.DataFrame({"SA1_CODE_2021": [], "value": []})
         assert result.to_dicts() == expected.to_dicts()
+
+
+# Sample data for tests
+data = {
+    "name": ["Alice", "Bob", "Charlie"],
+    "age": [25, 30, 35],
+    "city": ["New York", "Los Angeles", "Chicago"],
+}
+
+
+def filter_relevant_column(df: pl.LazyFrame, columns: list[str]) -> pl.LazyFrame:
+    """
+    Filter columns from a LazyFrame based on the provided column names.
+    If a column does not exist, it will be ignored.
+    """
+    existing_columns = [col for col in columns if col in df.columns]
+    return df.select(existing_columns)
```

Comment:
Why is this in here? redundant and this shouldn't be in `tests/` in the first place